config/database.yml

```diff
@@ -22,6 +22,7 @@ default: &default
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   host: postgres
   username: postgres
+  password: <%= ENV['POSTGRES_PASSWORD'] %>
```

Comment:
how about in a situation when you don't have password setup for postgres?

Reply:
then user should set one up 😄 
I am too keen on adding info as sensitive as password to the `database.yml` unless you want to convert to using `DATABASE_URL`